packages/es-components/src/components/containers/loading-skeleton/SkeletonTile.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import LoadingSkeleton from './LoadingSkeleton';
+
+const { Shape } = LoadingSkeleton;
+
+const TileContainer = styled(LoadingSkeleton)`
+  display: flex;
+  flex-direction: column;
+  margin: 17px 24px 0 14px;
+`;
+
+const HeaderFlex = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 20px;
+`;
+
+const IconPlaceholder = styled(Shape)`
+  height: 65px;
+  width: 65px;
+  flex-grow: 0;
+  margin-right: 40px;
+`;
+
+const TitlePlaceholder = styled(Shape)`
+  height: 26px;
+  flex-grow: 1;
+  align-self: center;
+`;
+
+const DescriptionPlaceholder = styled(Shape)`
+  height: 15px;
+  margin-bottom: 9px;
+`;
+
+const ButtonPlaceholder = styled(Shape)`
+  height: 17px;
```

Comment:
It'd be nice if the CSS properties could be alphabetized for consistency with our other components. It just helps combat duplicate properties and makes finding them a little easier while reading.